benchmark/src/compare-bfvrns-vs-bgvrns.cpp

```diff
@@ -60,119 +59,119 @@ usint mult_depth = 3;
+    // std::endl; std::cout << "log2 q = " <<
+    // log2(cc->GetCryptoParameters()->GetElementParams()->GetModulus().ConvertToDouble())
+    // << "\n" << std::endl;
+
+    return cc;
 }
 
 CryptoContext<DCRTPoly> GenerateBGVrnsContext(usint ptm) {
-  CCParams<CryptoContextBGVRNS> parameters;
-  parameters.SetMultiplicativeDepth(mult_depth);
-  parameters.SetPlaintextModulus(ptm);
-  parameters.SetKeySwitchTechnique(BV);
-  parameters.SetRescalingTechnique(FIXEDAUTO);
-
-  CryptoContext<DCRTPoly> cc = GenCryptoContext(parameters);
-  cc->Enable(PKE);
-  cc->Enable(KEYSWITCH);
-  cc->Enable(LEVELEDSHE);
-  cc->Enable(ADVANCEDSHE);
-
-  // std::cout << "\nParameters BGVrns for depth " << mult_depth << std::endl;
-  // std::cout << "p = " << cc->GetCryptoParameters()->GetPlaintextModulus() <<
-  // std::endl; std::cout << "n = " <<
-  // cc->GetCryptoParameters()->GetElementParams()->GetCyclotomicOrder() / 2 <<
-  // std::endl; std::cout << "log2 q = " <<
-  // log2(cc->GetCryptoParameters()->GetElementParams()->GetModulus().ConvertToDouble())
-  // << "\n" << std::endl;
-
-  return cc;
+    CCParams<CryptoContextBGVRNS> parameters;
+    parameters.SetMultiplicativeDepth(mult_depth);
+    parameters.SetPlaintextModulus(ptm);
+    parameters.SetKeySwitchTechnique(BV);
+    parameters.SetRescalingTechnique(FIXEDAUTO);
+
+    CryptoContext<DCRTPoly> cc = GenCryptoContext(parameters);
+    cc->Enable(PKE);
+    cc->Enable(KEYSWITCH);
+    cc->Enable(LEVELEDSHE);
+    cc->Enable(ADVANCEDSHE);
+
+    // std::cout << "\nParameters BGVrns for depth " << mult_depth << std::endl;
+    // std::cout << "p = " << cc->GetCryptoParameters()->GetPlaintextModulus() <<
+    // std::endl; std::cout << "n = " <<
+    // cc->GetCryptoParameters()->GetElementParams()->GetCyclotomicOrder() / 2 <<
+    // std::endl; std::cout << "log2 q = " <<
+    // log2(cc->GetCryptoParameters()->GetElementParams()->GetModulus().ConvertToDouble())
+    // << "\n" << std::endl;
+
+    return cc;
 }
 
 /*
  * BFVrns benchmarks
  */
 void BFVrns_EvalMultManyP2(benchmark::State& state) {
-  usint ptm = 2;
+    usint ptm = 2;
 
-  CryptoContext<DCRTPoly> cc = GenerateBFVrnsContext(ptm);
+    CryptoContext<DCRTPoly> cc = GenerateBFVrnsContext(ptm);
 
-  // KeyGen
-  KeyPair<DCRTPoly> keyPair = cc->KeyGen();
-  cc->EvalMultKeyGen(keyPair.secretKey);
+    // KeyGen
+    KeyPair<DCRTPoly> keyPair = cc->KeyGen();
+    cc->EvalMultKeyGen(keyPair.secretKey);
 
-  std::vector<int64_t> vectorOfInts = {1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
-  Plaintext plaintext = cc->MakeCoefPackedPlaintext(vectorOfInts);
+    std::vector<int64_t> vectorOfInts = {1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+    Plaintext plaintext               = cc->MakeCoefPackedPlaintext(vectorOfInts);
 
-  vector<Ciphertext<DCRTPoly>> ciphertexts;
-  for (int i = 0; i < (1 << mult_depth); i++)
-    ciphertexts.push_back(cc->Encrypt(keyPair.publicKey, plaintext));
+    vector<Ciphertext<DCRTPoly>> ciphertexts;
```

Comment:
why doesn't this require "std::"? I thought we removed the namespace up top

Reply:
aye, yaye, yaye.  Good catch.  Regex: [^:]vector has 677 hits in the codebase.